src/components/Book.jsx

```diff
@@ -1,63 +1,71 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { PropTypes } from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 import Button from './Buttons';
-import styleModule from '../styles/Book.module.css';
+import styles from '../styles/Book.module.css';
 
 const Book = ({
-  genre, title, author, status, chapter,
+  id, title, author, category,
 }) => {
-  const Btns = [
-    {
-      id: 1,
-      title: 'Comments',
-    },
-    {
-      id: 2,
-      title: 'Remove',
-    },
-    {
-      id: 3,
-      title: 'Edit',
-    },
-  ];
+  const dispatch = useDispatch();
+
+  const handleRemove = (bookId) => {
+    dispatch(removeBook(bookId));
+  };
```

Comment:
Good job so far :clap: Just one issue we can fix, please see that the remove action is not working, if i click on remove button nothing seems to happen, please fix this

[Screencast from 12-05-23 09:35:38 PM IST.webm](https://github.com/Metaverse-Nabeel/BookStore/assets/97521557/052af21c-9067-4293-9078-afdbbf38be4f)

Reply:
Hello @PinkMoon25 

Thank you so much for your kind feedback. I shall resolve the issue and will again ask for a review.

Regards